### PR TITLE
german translation

### PR DESCRIPTION
**Pull request by uDEV2019**
translated extension and webstore description, except the screenshots (for now)

### CODE REVIEW

File: src/locales/de.json.ts [hunk 1]
```diff
@@ -0,0 +1,332 @@
+import { exportAsMessages } from './helpers';
+
+exportAsMessages('_locales/en/messages.json', {
```
Comment (iorate) [inline src/locales/de.json.ts:3]: The filename should be `_locales/de/messages.json`.

**Review by iorate — CHANGES_REQUESTED** [review]
A lint error occurred. Run `yarn lintfix` to fix it.

You should add a dayjs locale for the German language. Open `src/scripts/dayjs-locales.ts` and add this line: `import 'dayjs/locale/de';`.

See also: https://github.com/iorate/uBlacklist#for-developers

File: src/locales/de.json.ts (hunk 1 above)
Comment (uDEV2019) [reply src/locales/de.json.ts]: Sorry, missed this.

**Comment by uDEV2019** [comment]
Hopefully i fixed my mistakes all

**Comment by iorate** [comment]
Would you run the following command and commit the change?
```shell
yarn lintfix
```

**Comment by iorate** [comment]
No, not like that. What I wanted to say is that you should run `yarn lintfix` LOCALLY on your machine, not on CI.

Please remove the last two commits. I will run `yarn lintfix` instead of you after merging your pull request.

**Comment by uDEV2019** [comment]
I'm not a developer.

**Pull request closed without merging**